lib/net/pool.js

```diff
@@ -462,6 +462,9 @@ class Pool extends EventEmitter {
    */
 
   async discoverSeeds(checkPeers) {
+    if (!this.discover)
```

Comment:
`this.options.discover`, there's no `this.discover` var in Pool (After last fix).